src/index.html

```diff
@@ -10,6 +10,7 @@
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
 <body class="mat-typography">
+  <img src="assets/branding/img/ndph-logo.png" />
```

Comment:
Should the name of the logo asset be a configuration option? We might want to change it per trial. But we could have a common fixed name (e.g. 'trial-logo.png'), or derive from the trial deployment name (e.g. 'mts451.png').

Reply:
I would go with 'logo.png'
we can always change in the future

Reply:
Changed to logo.png as suggested by Elad